utils/wp-cli-updatedeb.sh

```diff
@@ -31,8 +31,8 @@ Architecture: all
 Maintainer: Alain Schlesser <alain.schlesser@gmail.com>
 Section: php
 Priority: optional
-Depends: php5-cli (>= 5.6) | php-cli | php7-cli, php5-mysql | php5-mysqlnd | php7.0-mysql | php7.1-mysql | php7.2-mysql | php7.3-mysql | php7.4-mysql | php8.0-mysql | php8.1-mysql | php8.2-mysql | php-mysql, mysql-client | mariadb-client
```

Comment:
What's the significance of dropping all of these dependencies?

Reply:
1. Fix Lintian errors
2. Drop support for PHP 5.6 (EOL Dec 31, 2018)
3. Keep support for legacy PHP 7 thanks to Provides fields
4. Simplify dependencies to make it easier to maintain

Please note that I have revised one dependency (php-mysqlnd -> php-mysql | php-mysqlnd)

Reply:
Hmm not sure about dropping PHP 5.6 support for this package, as WP-CLI in fact still supports PHP 5.6. Could we restore that part somehow?

Reply:
@swissspidy, Yes ... commit 54056c0 restores PHP 5.6 support.
Downside is that Lintian reports the php-script-but-no-php-cli-dep error again.

WordPress 6.3 “Lionel” discontinued support for PHP 5 (Aug 8, 2023).
Are there any plans for WP-CLI to do the same?
If it is not imminent, I can create a lintian-override file.

Reply:
> Are there any plans for WP-CLI to do the same?

Not until Aug 8, 2024. See https://make.wordpress.org/cli/2019/01/15/wp-cli-php-requirements-strategy/